app/views/offers/show.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="container">
+  <h1> <%= @offer.user.first_name%>'s Loan</h1>
+</div>
+<div class="offer-img">
+  <img src="https://images2.minutemediacdn.com/image/upload/c_fill,g_auto,h_1248,w_2220/f_auto,q_auto,w_1100/v1555347972/shape/mentalfloss/istock_000045501888_medium.jpg" alt="" class="user-img" style="height: 30vh; width: 100vw; background-size: cover; object-fit: cover;">
```

Comment:
You can make it dynamic to whatever photo is going to be uploaded by the user in the offer creating process.

I think this can be used
`cl_image_tag current_user.photo`

However, if no photo is uploaded you should display a default or so...